src/components/Showcase/StoryList.js

```diff
@@ -58,7 +58,7 @@ class StoryList extends React.Component {
 
   componentDidMount() {
     Tabletop.init({
-      key: '1aHn_C0LjMAib1X1m2kfuI3YvD5zzvi7cqbaymxXbXZs',
+      key: '1I2nTG_lst4nYrg8z1e7RaolC16A-M7f_lO_zRaV9L5s',
```

Comment:
Did something happen to the key?

Reply:
@kilemensi Yap. Noticed this last week. The showcase stories were not showing, so I did a check only to find out that we were using the `SeaSensors` key instead of the `sensors.AFRICA` gsheet key. Made an update, it should be fine now